src/api/odometry/odomMath.cpp

```diff
@@ -44,4 +44,15 @@ double OdomMath::computeDistance(double xDiff, double yDiff) {
 double OdomMath::computeAngle(double xDiff, double yDiff, double theta) {
   return std::atan2(yDiff, xDiff) - theta;
 }
+
+QAngle OdomMath::constrainAngle360(QAngle theta) {
+  double degreeValue = theta.convert(degree) - 360.0 * std::floor(theta.convert(degree) / 360.0);
+  return QAngle(degreeValue * degreeToRadian);
```

Comment:
Please don't rely on `QAngle` storing its internal value in radians. You can just do `degreeValue * degree`. (here and below)

Reply:
Done